drake/core/Core.h

```diff
@@ -70,25 +130,81 @@ namespace Drake {
     };
     std::size_t size() { return vec1.size()+vec2.size(); }
 
+    friend Eigen::Matrix<ScalarType,Vector1<ScalarType>::RowsAtCompileTime + Vector2<ScalarType>::RowsAtCompileTime,1> toEigen(const CombinedVector<ScalarType,Vector1,Vector2>& vec) {
+      Eigen::Matrix<ScalarType,Vector1<ScalarType>::RowsAtCompileTime + Vector2<ScalarType>::RowsAtCompileTime,1> x;
```

Comment:
This doesn't work when either Vector1 or Vector2 has dynamic size at compile time.